sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -71,7 +71,16 @@ object JdbcUtils extends Logging with SQLConfHelper {
       } finally {
         statement.close()
       }
-    }.isSuccess
+    } match {
+      case Success(_) => true
+      // This may be a known `TableAlreadyExist` exception, which is normal and proves that the
+      // table does not exist, but perhaps some other exception, which in fact does exist, will
+      // cause an error when renewing the table later. We need to know about the exception stack
```

Comment:
will end users see the error eventually when the query triggers table lookup?

Reply:
No, end users see `warn` level only.

Reply:
I think we should deal with the third state.